extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzBuildTimeConfig.java

```diff
@@ -21,6 +21,8 @@ public class QuartzBuildTimeConfig {
 
     /**
      * The frequency (in milliseconds) at which the scheduler instance checks-in with other instances of the cluster.
+     * <p>
+     * Ignored if using a `ram` store.
```

Comment:
We could replace the `ram` with `{@link io.quarkus.quartz.runtime.StoreType#RAM}`... Also I think that javadoc is used by default and for asciidoc you need to add `@asciidoclet`, e.g. https://github.com/quarkusio/quarkus/blob/9b2ee9e99aeef41c6ab1a4dcc5c0e39a2dea422e/extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfig.java#L26.

Reply:
Thanks for the suggestion. This https://quarkus.io/guides/quartz#quarkus-quartz_quarkus.quartz.table-prefix is how what I've will render. 
Are you suggesting that we add a link to https://github.com/quarkusio/quarkus/blob/fbbdac541e1fe9db104f370ae6494d83d2e7f329/extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/StoreType.java#L4 and have the javadoc inform of asciidoc for proper rendering? 

I am afraid a link like `{@link io.quarkus.quartz.runtime.StoreType#RAM}` won't be understood by the doc generator as it only treats known links (vertx classes, java classes) when rendering javadoc. And as asciidoc, it'll just render the content as it since `{@link ..}` isn't correct asciidoc format.

Reply:
It does not have to be a link but I think that we should reference the enum type somehow...

Reply:
Okay, makes sense. We've the enum type in the `store-type` config item, I'll  add `{@link io.quarkus.quartz.runtime.StoreType#RAM}` to the javadoc for the rest of the configurations where we reference `ram` store type.

Reply:
done